astropy/utils/iers/iers.py

```diff
@@ -802,3 +825,250 @@ def _substitute_iers_b(cls, table):
             table['dY_2000A_B'][:n_iers_b] = iers_b['dY_2000A']
 
         return table
+
+
+class LeapSeconds(QTable):
+    """Leap seconds class, holding TAI-UTC differences.
+
+    The table should hold columns 'year', 'month', 'tai_utc'.
+
+    Methods are provided to initialize the table from IERS ``Leap_Second.dat``,
+    IETF/ntp ``leap-seconds.list``, or built-in ERFA/SOFA.
```

Comment:
The seems like a likely gotcha for potential users...

```suggestion
    IETF/ntp ``leap-seconds.list``, or built-in ERFA/SOFA. Note that to
    actually update the leap seconds used by ``astropy``, the 
    `update_erfa_leap_seconds` method must be explicitly called.
```

Reply:
Done a bit differently, without an explicit name.  Could rewrite this part to enumerate all class methods, but that seemed excessive.